blogposts/2021/creating-sourcegraph-product-design-principles.md

```diff
@@ -0,0 +1,132 @@
+
+Finally, we shifted back to an asynchronous workflow to cast anonymous votes for the themes that resonated most strongly with each of us as individuals.
+
+
+#### 4. Continuation and refinement
+
+One of the trickiest parts of a collective activity is bridging the gap between collective work and cohesive outcomes. 
+To keep us moving forward, one member of our team took the outcomes of the workshop—the themes, the notes from our discussion, and the voting—and wrote a first draft of our potential product design principles. 
+This artifact became the basis for further continuation and refinement.
+
+
+#### 5. Sharing and collecting feedback from the whole organization
+
+Once we all felt strongly that our product design principles captured our team’s vision and values, we shared our proposed principles as an [RFC](https://docs.google.com/document/d/1zRbtZR68ZITYypSAJJ63Ir_fFPxJfTtidJmsrxUXW7o/edit#) with everyone at Sourcegraph, which you can now read too.
+
+
+## Sourcegraph’s product design principles
+
+*   **A personal tool within a larger workflow**
+
+Sourcegraph is a powerful yet personal tool that exists within a larger workflow. Design for familiar patterns with thoughtful defaults, while embracing personalization and adaptability.
+
+*   **Made for everyone**
+
+Our purpose is to make it so [everyone can code](/company/strategy#purpose). This demands we make Sourcegraph accessible and useful for all developers through universal, inclusive design.
+
+*   **Gracefully manage complexity**
+
+Sourcegraph supports complex product requirements, but also empowers users to manage this complexity for their individual needs.
+
+*   **Code as content**
+
+More time is spent reading than writing code. Elevate the craft of code as content.
+
+*   **Trust is earned**
+
+Sourcegraph is the source of truth for developers working on and learning about their code, but this trust is earned. Accuracy, transparency, recency, and honesty together work to uphold this source of truth.
+*   **Create momentum**
+
+Help developers create and maintain flow. To do this, Sourcegraph must be fast in every way. We design purposefully to help users iterate and build momentum.
+
+We’ve published our [product design principles](https://about.sourcegraph.com/handbook/product/design_principles) in our open handbook.
+
+## The path forward
+
+Now that our team has these product design principles, we’re using them in our work to make better decisions faster. 
+Over time, our team and developers using Sourcegraph in their work will see these principles reflected as these decisions accumulate.
+
+If your company doesn’t have product design principles, consider creating them together! 
+Sourcegraph is an open company, and so our process, artifacts, and resources are all available to you.
+
+**Resources**
+
+*   [Design principles planning and workshop document](https://docs.google.com/document/d/10Ko8z1ozVaUStuPG2-cM_EJUVJvcgabv8qZYNAJXn24/edit#)
```

Comment:
@quinnkeast Can you update the permissions here so people outside of Sourcegraph can view?

Reply:
Updated!